app/scripts/lib/channels/notifier.js

```diff
@@ -51,22 +80,31 @@ define(function (require, exports, module) {
       }
     },
 
-    triggerAll: function (event, data, context) {
-      this.triggerRemote(event, data);
-      this.trigger(event, data, context);
+    triggerAll: function (command, data, context) {
+      this.triggerRemote(command, data);
+      this.trigger(command, data, context);
     },
 
-    triggerRemote: function (event, data) {
+    triggerRemote: function (command, data) {
+      data = eliminateUndefinedProperties(data);
```

Comment:
I can deal with this. Can you add a note about why? The reason I noted is because not all payloads are actually serialized, in those cases the keys with undefined values are sent to the receiver.